src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,13 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts persons in list by name in alphabetical order.
+     */
+    public void sortByName() {
+        internalList.sort(Comparator.comparing(person -> person.getName().fullName));
```

Comment:
Should we be sorting names in a case-sensitive manner?

Reply:
Good point. A case-insensitive sort would be more appropriate considering the use case (quickly finding contacts).

Do you agree?

Reply:
Yes! That's certainly a quality of life improvement.

Reply:
Done.